chromadb/server/fastapi/__init__.py

```diff
@@ -352,9 +359,33 @@ def get_tenant(self, tenant: str) -> Tenant:
         ),
     )
     def list_collections(
-        self, tenant: str = DEFAULT_TENANT, database: str = DEFAULT_DATABASE
+        self,
+        tenant: str = DEFAULT_TENANT,
```

Comment:
nit, make tenant and database secondary

Reply:
secondary? what do you mean?

Reply:
friendly bump here :)

Reply:
Just means to reorder the parameters to make tenant, database last to match the remaining functions.